app/covidbed/model/orga.py

```diff
@@ -49,7 +49,7 @@ class Region(Base):
 
     code = db.Column(db.String(2),  primary_key=True)
     tncc = db.Column(db.String(20), nullable=False)
-    libelle = db.Column(db.String(20), nullable=False)
+    libelle = db.Column(db.String(30), nullable=False)
```

Comment:
A migration state has to be created manually, because sqlalchemy doesn't detect it

Reply:
migration added